### PR TITLE
[FCE-621] Add jitpack cache warmup

### PR DESCRIPTION
**Pull request by mironiasty**
## Description

When we publish new release (tag), we have to push libraries to CocoaPods and npm. But jitpack works another way, and it pulls our code from repo.

This PR is adding step to let know Jitpack that new version was released.

This PR also adds formatting for .github files.

## Motivation and Context

Right now, when we publish new version, first user that will try to build app will get error - as Android won't work. There are few minutes required after first request for JitPack to cache everything. With this change, it should work right away.

## How has this been tested?

To fully test this addition, new version has to be released. But I should not break any existing flow.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to
      not work as expected)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -26,11 +26,18 @@ jobs:
...
+          registry-url: "https://registry.npmjs.org"
       - run: yarn
       - run: yarn prepare
-      - run: yarn npm publish --access public 
+      - run: yarn npm publish --access public
         working-directory: packages/react-native-client
         env:
           NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
+  publish-android-client:
+    # actually it just warmup jitpack cache, so android client is available sooner
```
Comment (karkakol) [inline .github/workflows/release.yml:38]: **question:** 
What does this comment mean?

**Review by karkakol — COMMENTED** [review]
If i understand properly it tries to fetch released version from jitpack and this triggers build so it is available faster?

File: .github/workflows/release.yml (hunk 1 above)
Comment (mironiasty) [reply .github/workflows/release.yml:38]: >If i understand properly it tries to fetch released version from jitpack and this triggers build so it is available faster?

That is correct
 
>What does this comment mean?

This means that this is not `publish` action, but just ask jitpack to build/cache package. I will think on how to phrase it better

**Review by MiloszFilimowski — APPROVED** [review]

File: .github/workflows/release.yml (hunk 1 above)
Comment (karkakol) [reply .github/workflows/release.yml:38]: I think that sth like this:
```
Packages built by Jitpack are not immediately available upon the creation of a tag or branch. Instead, availability occurs when an initial attempt is made to download the specified tag or branch. This mechanism ensures that Jitpack only builds a package when it is first requested. This approach efficiently handles the generation of newly released versions.
```

Comment (mironiasty) [reply .github/workflows/release.yml]: I've updated comment

**Review by MiloszFilimowski — APPROVED** [review]

**Review by karkakol — APPROVED** [review]
taco_hulk_approves

**Pull request merged**